docs/guides/how_to_guides/creating_and_editing_expectations/how_to_create_an_expectation_suite_with_the_user_configurable_profiler.rst

```diff
@@ -242,9 +242,369 @@ This guide will help you create a new Expectation Suite by profiling your data w
+                    metric_value_kwargs:
+                      quantiles:
+                        - 0.05
+                        - 0.25
+                        - 0.50
+                        - 0.75
+                        - 0.95
+                  p_values:
+                    min_value: ($false_positive_threshold / 2)
+                    max_value: 1 - ($false_positive_threshold / 2)
+                - parameter_name: my_parameter_max
+                  class_name: MetricParameterBuilder
+                  metric_name: column.max
+                  metric_domain_kwargs: $domain.domain_kwargs # QUESTION: Should this be optional and inferred?
+                - id: my_parameter_quantiles
+                  class_name: MetricParameterBuilder
+                  metric_name: column.quantile_values
+                  metric_domain_kwargs: $domain.domain_kwargs # QUESTION: Should this be optional and inferred?
+                  metric_value_kwargs:
+                    quantiles:
+                      - 0.05
+                      - 0.25
+                      - 0.50
+                      - 0.75
+                      - 0.95
+              configuration_builders:
+                - expectation: expect_column_min_to_be_between
+                  min_value: $my_parameter_false_positive_threshold_min.min_value
+                  max_value: $my_parameter_false_positive_threshold_min.max_value
+                - expectation: expect_column_quantile_values_to_be_between
+                  quantiles:
+                    - 0.05
+                    - 0.25
+                    - 0.50
+                    - 0.75
+                    - 0.95
+                  value_ranges: $my_parameter_quantile_ranges
+                - expectation: expect_column_max_to_be_between
+                  min_value: 0.9 * $my_parameter_max
+                  max_value: 1.1 * $my_parameter_max
+                - expectation: expect_column_median_to_be_between
+                  min_value:  # in this example, the resulting configuration is not a value, but an evaluation parameter referring to a metric that must be calculated for each validation to validation of the configuration (QUESTION: can this be performed by a parameter definition relative to the current batch instead of Evaluation Parameter?)
+                    "$PARAMETER": 0.9 *
+                       metric_name: column.median
+                       metric_batch_kwargs:
+                         batch_request:
+                           partition_request:
+                             partition_index: -1
+                         column: $domain.domain_kwargs.column # QUESTION: Should this be optional and inferred?
+                  max_value:  # in this example, the resulting configuration is not a value, but an evaluation parameter refering to a metric that must be calculated for each validation to validation of the configuration
+                    "$PARAMETER": 1.1 *
```

Comment:
It seems that this is repeated here - if there is no number at the end of the multiplication or the * doesn't specify multiplication, could you explain why that is?